src/gateway_config_sup.erl

```diff
@@ -32,8 +32,6 @@ start_link() ->
 %% Child :: {Id,StartFunc,Restart,Shutdown,Type,Modules}
 init([]) ->
     {ok, B} = ebus:system(),
-    %% Set up this connman to be the agent for authorization requests
-    ok = connman:start_agent(),
```

Comment:
Do you also need to call connman:start_agent() before connman:enable(wifi, true) in gateway_gatt_service.erl?

Reply:
No, I think that would make it take longer again.The agent only really needs to be there when the user is trying to auth to the network.